frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { testName, checkErrors } from '../../support';
+import {
+  SnapshotDetails,
+  snapshotClassDropdown,
+  dropdownFirstItem,
+  pvcDropdown,
+} from '../../views/snapshot';
+import { listPage } from '../../views/list-page';
+import { PVC, testerDeployment, SnapshotClass } from '../../mocks/snapshot';
+import { detailsPage } from '../../views/details-page';
+import { submitButton } from '../../views/form';
+
+const snapshotName = `${PVC.metadata.name}-snapshot`;
+
+/**
+ * This suite works only with AWS platform.
+ */
+
+describe('Snapshot Tests', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+    cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc apply -f -`);
+    // Let the PVC get bound
+    cy.wait(10 * 1000);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it('Creates Snapshot', () => {
+    cy.clickNavLink(['Storage', 'Volume Snapshots']);
+    cy.location('pathname', { timeout: 60000 }).should(
+      'include',
+      `snapshot.storage.k8s.io~v1beta1~VolumeSnapshot`,
+    );
+    listPage.clickCreateYAMLbutton();
+    cy.get(pvcDropdown).click();
+    cy.get(dropdownFirstItem)
+      .first()
+      .click();
+    cy.get(snapshotClassDropdown).click();
+    cy.get(dropdownFirstItem)
+      .first()
+      .click();
+    cy.get(submitButton).click();
+    cy.location('pathname', { timeout: 60000 }).should(
+      'include',
+      `snapshot.storage.k8s.io~v1beta1~VolumeSnapshot/${PVC.metadata.name}-snapshot`,
+    );
+    detailsPage.titleShouldContain(PVC.metadata.name);
+    // eslint-disable-next-line promise/catch-or-return
+    cy.exec(`oc get VolumeSnapshot ${PVC.metadata.name}-snapshot -n ${testName} -o json`)
```

Comment:
Instead of adding `// eslint-disable-next-line promise/catch-or-return`, you can just `return cy.exec(...`

Reply:
You should no longer need these eslint-disable rules, recently [udpated](https://github.com/openshift/console/pull/6398/files#diff-8949092bfde58c674fd96f762a30519b)